processor/src/main/java/com/linecorp/decaton/processor/runtime/SubscriptionBuilder.java

```diff
@@ -137,7 +139,18 @@ public ProcessorSubscription build() {
         DecatonProcessorSupplier<byte[]> retryProcessorSupplier = null;
         if (retryConfig != null) {
             Properties producerConfig = Optional.ofNullable(retryConfig.producerConfig())
-                                                .orElse(consumerConfig);
+                                                .orElseGet(() -> {
+                                                    Properties producerProps = new Properties();
+                                                    Set<String> definedProps = ProducerConfig.configNames();
+                                                    consumerConfig.entrySet()
```

Comment:
I know it's weird but I think it is encouraged to not to rely on `Hashtable`'s methods when using `Properties` because most of them are not type-safe (`entrySet()` returns `Map<Object, Object>`). 
I think it's better to iterate over `stringPropertyNames()` with `get/setProperty` even for this trivial case.